container/common/fsHandler.go

```diff
@@ -106,31 +106,30 @@ func (fh *realFsHandler) update() error {
 }
 
 func (fh *realFsHandler) trackUsage() {
-	fh.update()
 	longOp := time.Second
 	for {
+		if err := fh.update(); err != nil {
+			klog.Errorf("failed to collect filesystem stats - %v", err)
+			fh.period = fh.period * 2
+			if fh.period > maxBackoffFactor*fh.minPeriod {
+				fh.period = maxBackoffFactor * fh.minPeriod
+			}
+		} else {
+			fh.period = fh.minPeriod
+		}
+		start := time.Now()
```

Comment:
start needs to be defined before the update.  The current change makes duration ~= 0 always.

Reply:
@dashpole 
Corrected.